services/web/server/src/simcore_service_webserver/exporter/archiving.py

```diff
@@ -25,12 +25,28 @@ def _strip_directory_from_path(input_path: Path, to_strip: Path) -> Path:
     return Path(str(input_path).replace(to_strip, ""))
 
 
+def _read_in_chunks(file_object, chunk_size=1024 * 8):
+    """Lazy function (generator) to read a file piece by piece.
+    Default chunk size: 8k."""
+    while True:
+        data = file_object.read(chunk_size)
+        if not data:
```

Comment:
MINOR: being picky ... I am a bit suspicious about ``if not data`` condition here. I wonder if this could return a zero , i.e. ``data==0`` . would ``if not len(data)``?
An alternative
```python
more_chunk = True
while more_chunk:
     chunk = file_object.read(chunk_size)
     more_chunk = len(chunk) == chunk_size
     yield chunk
```

Reply:
I have checkups in place for this. Never saw an issue here (and never saw it before)